common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/interactors/ResetPasswordInteractor.kt

```diff
@@ -94,11 +103,29 @@ class ResetPasswordInteractor(
             headers,
             encodedRequest.toByteArray(charset(ObjectMapper.ENCODING_SCHEME))
         )
+
         val apiResponse = nativeAuthResponseHandler.getResetPasswordStartApiResponseFromHttpResponse(
             requestCorrelationId = requestCorrelationId,
             response = httpResponse
         )
-        return apiResponse.toResult()
+
+        Logger.infoWithObject(
+            "${TAG}.rawResponseToResetPasswordStartApiResult",
+            apiResponse.correlationId,
+            "rawApiResponse = ",
+            apiResponse
+        )
+
+        val result = apiResponse.toResult()
+
+        Logger.infoWithObject(
```

Comment:
This is a good idea. Did you test this? Does this not produce too many logs? If you're fine with this then so am I.

Reply:
I think this change also comes from the idea that remove the logs from OAuth2Strategy, RequestProvider and RequestHandler, but have some logging in the API layer. 
In my opinion and experience, they are pretty useful and direct. Meanwhile, I am wondering if this batch of logs would cause some side effect of the SDK, for example, size?
Does Logger.debug or Logger.verbose help? Currently, we have infoWithObject and warnWithObject.

Reply:
It won't affect the binary size, but might affect performance. I doubt it though, and I don't think we need to prematurely optimise for that.